iroh-bytes/src/blobs.rs

```diff
@@ -26,6 +26,11 @@ impl Collection {
         })
     }
 
+    /// Serialize this collection to a std Vec<u8>
+    pub fn to_bytes(&self) -> Result<Vec<u8>> {
```

Comment:
`as_bytes`? `to_` usually takes `self`

Reply:
I think into_... takes self. It seems OK according to the conventions. Although I also never really know in some cases...

![image](https://github.com/n0-computer/iroh/assets/248257/6b770f3d-68cf-4a24-ae19-288b3fbbfe76)

This is an expensive conversion, borrowed -> owned, so to_seems appropriate.

Reply:
👍